### PR TITLE
[Infra] Update ONE-vscode version to v0.5.0

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit updates ONE-vscode version to v0.5.0.
It also updates release note for v0.5.0.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

File: infra/release/release_note.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
 ## ONE-vscode
 
 ### ONE Explorer View
 
-- Improve latency on startup/refresh/file system changes
-- Revive rename command for ONE configuration files
-- Support refactoring - a function that automatically updates corresponding fields in ONE configuration files when parent model files are renamed
-- Support multi-root workspace
+- Open tflite and onnx files in CircleViewer
+  - Circle Schema update to v0.6
```
Comment (dayo09) [inline infra/release/release_note.md:8]:
```suggestion
  - Update Circle Schema to v0.6
```

File: infra/release/release_note.md [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+- Improvement ONE Explorer file viewer according to the file type
```
Comment (dayo09) [inline infra/release/release_note.md:9]:
```suggestion
- Improved ONE Explorer file viewer according to the file type
```

File: infra/release/release_note.md [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
 
 ## ONE-vscode
 
 ### ONE Explorer View
 
-- Improve latency on startup/refresh/file system changes
-- Revive rename command for ONE configuration files
-- Support refactoring - a function that automatically updates corresponding fields in ONE configuration files when parent model files are renamed
-- Support multi-root workspace
+- Open tflite and onnx files in CircleViewer
```
Comment (dayo09) [inline infra/release/release_note.md:7]:
```suggestion
- Support tflite and onnx files with CircleViewer
```

File: infra/release/release_note.md [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
+  - Circle Schema update to v0.6
+- Improvement ONE Explorer file viewer according to the file type
+  - Open json file with default text editor
+- Support keybindings
+  - rename: F2
+  - delete: DEL
+  - run cfg: ctrl + F7
+
 ### Toolchain View
```
Comment (dayo09) [inline infra/release/release_note.md:16]:
```suggestion
```

File: infra/release/release_note.md [hunk 5]
```diff
@@ -1,14 +1,21 @@
...
+- Support tflite and onnx files with CircleViewer
+  - Updated Circle Schema to v0.6
+- Improvement ONE Explorer file viewer according to the file type
+  - Open json file with default text editor
+- Support keybindings
+  - rename: F2
+  - delete: DEL
+  - run cfg: ctrl + F7
 
-- Support ONE toolchain
```
Comment (jyoungyun) [inline infra/release/release_note.md:16]:
```suggestion
```

File: infra/release/release_note.md [hunk 6]
```diff
@@ -1,14 +1,20 @@
...
+### Device View
 
+- Remove the HostPC target so that only the host and remote are visiable
```
Comment (glistening) [inline infra/release/release_note.md:18]:
```suggestion
- Remove the HostPC target so that only the host and remote are visible
```

**Review by glistening — APPROVED** [review]
LGTM

**Review by dayo09 — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**